twisted/spread/flavors.py

```diff
@@ -107,6 +109,10 @@ def remoteMessageReceived(self, broker, message, args, kw):
         """
         args = broker.unserialize(args)
         kw = broker.unserialize(kw)
+
+        if not isinstance(message, str):
+            message = message.decode('utf8')
```

Comment:
this code is not covered ... and I find it hard to understand why we got it here.

I am expected that remote messages are always received as bytes... so they should always be decoded.

Can we have the `message` argument documented to make it clear what types are accepted ?